src/applications/validate-mhv-account/containers/VerifyIdentity.jsx

```diff
@@ -13,7 +13,7 @@ const VerifyIdentity = () => {
           you access to your personal health information.
         </p>
         <button
-          onClick={verify}
+          onClick={() => verify({ returnURL: '/my-health-account-validation' })}
```

Comment:
Rather than modifying only one of the redirect functions to accept an optional param to force a return URL, and possibly having to follow suit with the other redirects for consistency, I think it would be better to expect the redirect to come back to this page, but redirect to where the user needs to be if they are already verified. And verified users in general should not have to visit this page.